### PR TITLE
Add Lava to Chain Registry

### PR DESCRIPTION
**Pull request by KagemniKarimu**
- Mkdir ./lava
- Create ./lava/assetlist.json
- Create  ./lava/chain.json
- Mkdir ./lava/images/
- Add lava-icon.png  lava-icon.svg to ./lava/images

### CODE REVIEW

File: lava/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "lava-testnet-1",
```
Comment (JeremyParish69) [inline lava/assetlist.json:3]: please change to just 'lava'

File: lava/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "exponent": 6
+          }
+        ],
+        "base": "ulava",
+        "name": "LAVA",
+        "display": "lava",
+        "symbol": "LAVA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.svg"
```
Comment (JeremyParish69) [inline lava/assetlist.json:23]: One of these images has a black background and the other is transparent. Which is preferred? Can we make one of them an alternative kept in the folder but not referenced here?

File: lava/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        },
+        "coingecko_id": "lava",
```
Comment (JeremyParish69) [inline lava/assetlist.json:25]: I don't think this id is used by CoinGecko for lava... It seems to be for lavaswap

File: lava/chain.json [hunk 4]
```diff
@@ -0,0 +1,209 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "lava-testnet-1",
+    "status": "live",
+    "network_type": "testnet",
```
Comment (JeremyParish69) [inline lava/chain.json:5]: since this is a testnet, can we please move all of this into the /testnets/ directory?

File: lava/chain.json [hunk 5]
```diff
@@ -0,0 +1,209 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "lava-testnet-1",
```
Comment (JeremyParish69) [inline lava/chain.json:3]: chain_name -> lavatestnet

Comment (KagemniKarimu) [reply lava/chain.json]: Corrected:
Commit [636c8b3](https://github.com/cosmos/chain-registry/pull/2244/commits/636c8b3fcdf2b38274413e74b426dc3bc982584f)

File: lava/chain.json (hunk 4 above)
Comment (KagemniKarimu) [reply lava/chain.json]: Corrected:
[cfa6968](https://github.com/cosmos/chain-registry/pull/2244/commits/cfa6968697836fa1bd0e2d5cf47750789ca1284a)

File: lava/assetlist.json (hunk 3 above)
Comment (KagemniKarimu) [reply lava/assetlist.json]: Corrected: 
[6bd2433](https://github.com/cosmos/chain-registry/pull/2244/commits/6bd24332679b38d296fc07f4a9ddb3693f139f2b)

File: lava/assetlist.json (hunk 1 above)
Comment (KagemniKarimu) [reply lava/assetlist.json]: Corrected:
[dae51fa](https://github.com/cosmos/chain-registry/pull/2244/commits/dae51fa9ed2a0b9e92a8e6dbd1a6de701cc726e1)

**Comment by KagemniKarimu** [comment]
All suggested changes have been corrected! Ready for review.

**Pull request merged**